debug_toolbar/utils.py

```diff
@@ -35,7 +35,20 @@ def get_module_path(module_name):
         raise ImproperlyConfigured(
             'Error importing HIDE_IN_STACKTRACES: %s' % (e,))
     else:
-        source_path = inspect.getsourcefile(module)
+        # Note that source_path is a byte string in the default
+        # encoding for the filesystem, which means that
+        # calling .endswith() will attempt to convert it to
+        # ASCII.
+        #
+        # os.path.dirname() is OK dealing with byte strings
+        # in unknown encodings, but .endswith() is not, so
+        # here we cast it down to ASCII for the __init__.py
+        # check, since all we care about is if we need to
+        # call os.path.dirname() on it.
+        smashed_to_ascii = source_path.decode('ascii', 'replace').encode(
```

Comment:
`source_path` is undefined, the line that defines it was removed by this commit (l. 38)!